modular_skyrat/modules/cortical_borer/code/cortical_borer_egg.dm

```diff
@@ -0,0 +1,74 @@
+/obj/effect/mob_spawn/ghost_role/borer_egg
+	name = "borer egg"
+	desc = "An egg of a creature that is known to crawl inside of you, be careful."
+	icon = 'modular_skyrat/modules/cortical_borer/icons/animal.dmi'
+	icon_state = "brainegg"
+	layer = BELOW_MOB_LAYER
+	density = FALSE
+	mob_name = "cortical borer"
+	///Type of mob that will be spawned
+	mob_type = /mob/living/simple_animal/cortical_borer
+	role_ban = ROLE_DRONE
```

Comment:
Borers are an antag, they should be using a proper antagonist ban type (like xeno)